### ISSUE TITLE
PDF rendering error

### ISSUE THREAD

**Issue by tariqul-islam** [mozilla/pdf.js#18310]
Attach (recommended) or Link to PDF file here:
Happens wit all the PDFs.

Configuration:
- Web browser and its version:  Firefox 127.0 
        [Problem was 
- Operating system and its version:

RELEASE=20.1
CODENAME=ulyssa
EDITION="MATE"
DESCRIPTION="Linux Mint 20.1 Ulyssa"
DESKTOP=Gnome
TOOLKIT=GTK 

- PDF.js version: 4.3.8 [c419c8333]

- Is a browser extension: No

Steps to reproduce the problem:
1. Opening any PDF with the browser.
2.

What is the expected behavior? (add screenshot)
White background, legible fonts.

What went wrong? (add screenshot)
![Screenshot at 2024-06-20 22-06-48](https://github.com/mozilla/pdf.js/assets/4572205/47b413a0-fc9e-40d8-9a5f-d4643d74c7f0)

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):

**Comment by timvandermeij** [comment]
If this happens with every PDF it's most likely an issue in the browser and not in PDF.js, so it'll likely not be possible to fix it here. Could you try starting Firefox with a fresh profile, see https://support.mozilla.org/en-US/kb/refresh-firefox-reset-add-ons-and-settings, since this could be caused by e.g. browser settings that are different from the defaults? Otherwise, please try updating graphics drivers or toggling hardware acceleration as those can also be causes of rendering defects.

**Comment by jwoertink** [comment]
This happens to me too. One thing I noticed though is it starts happening after I've had Firefox running for a while. For example, yesterday I couldn't open a single PDF. Every single one just would render a black screen with blue boxes where text or input fields were. Then this morning I closed the browser and reopened, and now I can see all of those same PDFs. I'll update this comment with a screenshot once it starts happening again. I'm running `127.0 (64-bit)` Linux

**Comment by periodic** [comment]
I had this same issue and can confirm that restarting Firefox fixed it for me.  All PDFs were having these rendering artifacts just like the screenshots above (various manuals, a paper from Arxiv, documents from my health provider).

Browser: 126.0.1 (what I first upgraded to) / 127.0 (current)
Install Type: dpkg from repositories
OS: Pop!_OS 22.04 LTS
Windowing System: X11
GNOME Version: 42.9
Graphics: NVIDIA 550.67

Is it possible that I was on an older version and restarting updated it?  I did another system upgrade and that didn't seem to harm anything.  I'll report back if it starts happening again.
